### PR TITLE
Update docs for configuring geo location based statistics

### PR DESCRIPTION
**Pull request by ChamathNS**
## Purpose
$subject
Fixes https://github.com/wso2/product-is/issues/12011

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/learn/enabling-geolocation-based-statistics.md [hunk 1]
```diff
@@ -3,11 +3,9 @@
 Follow the steps below to configure WSO2 IS Analytics Server to display
 the regions of the users authenticated though WSO2 Identity Server.
 
-1.  Download the geolocation data from
-    [here](https://svn.wso2.org/repos/wso2/people/tharindua/geodata/Geolocation%20Data.tar.gz)
+1.  Use the Geo Location dataset that you created [here](../../learn/creating-geo-location-data-set/) when configuring geo location based statistics.
```
Comment (ashendes) [inline en/docs/learn/enabling-geolocation-based-statistics.md:6]: Better to say "Create a Geo Location dataset by following the guide given [here]"

@gomathyk Please verify

File: en/docs/learn/creating-geo-location-dataset.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Creating Geo Location Data Set
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:1]:
```suggestion
# Creating a Geolocation Dataset
```

File: en/docs/learn/creating-geo-location-dataset.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Follow the procedure below in order to create a Geo Location dataset to use when [Enabling Geolocation Based Statistics
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:3]:
```suggestion
Follow the procedure below in order to create a geolocation dataset to use when [Enabling Geolocation Based Statistics
```

File: en/docs/learn/creating-geo-location-dataset.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+](../../learn/enabling-geolocation-based-statistics).
+
+1.  Create an account in [www.maxmind.com](https://www.maxmind.com/) and download the **GeoLite2 City: CSV Format** as a ZIP file.
+2.  Download the geoip-2-csv-converter from <https://github.com/maxmind/geoip2-csv-converter/releases> according to your operating system.
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:7]:
```suggestion
2.  Download the geoip-2-csv-converter from <https://github.com/maxmind/geoip2-csv-converter/releases> depending on your operating system.
```

File: en/docs/learn/creating-geo-location-dataset.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+#### Prepare the database entries
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:9]:
```suggestion
## Prepare the database entries
```

File: en/docs/learn/creating-geo-location-dataset.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+1.  Unzip the latest CSV file and the geoip-2-csv-converter you have downloaded in the steps above.
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:11]:
```suggestion
1.  Unzip the latest CSV file and the geoip-2-csv-converter you have downloaded in the previous step.
```

File: en/docs/learn/creating-geo-location-dataset.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+2.  Run `update-geolocation-data.sh` file using the command below.
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:12]:
```suggestion
2.  Run the `update-geolocation-data.sh` file using the command below.
```

File: en/docs/learn/enabling-geolocation-based-statistics.md (hunk 1 above)
Comment (ChamathNS) [reply en/docs/learn/enabling-geolocation-based-statistics.md]: Thanks. Addressed with https://github.com/wso2/docs-is/pull/2716/commits/ad42ac1b9834f3a3ac2e418711faf1986dae00db

File: en/docs/learn/creating-geo-location-dataset.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+    For Linux  : [update-geolocation-data.sh](../../assets/attachments/learn/geo-location/linux/update-geolocation-data.sh)
+    
+    For Mac: [update-geolocation-data.sh](../../assets/attachments/learn/geo-location/mac/update-geolocation-data.sh)
+
+    ```shell
+    sh update-geolocation-data.sh
+    ```      
+
+    -   Enter the path to the extracted GeoLite2-City-Blocks-IPv4 directory which you downloaded first, as the response for **Enter path to GeoLite2-City-Blocks-IPv4 directory:**
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:22]: Please make this a part of the numbered list.

File: en/docs/learn/creating-geo-location-dataset.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+        E.g :   /&lt;PATH\_TO&gt;/GeoLite2-City-CSV_20200310
+
+    -   Enter the path to geoip2-csv-converter directory as the response for **Enter path to geoip2-csv-converter home directory:**
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:26]:
```suggestion
    -   Enter the path to the `geoip2-csv-converter` directory as the response for **Enter path to geoip2-csv-converter home directory:**
```

File: en/docs/learn/creating-geo-location-dataset.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+        E.g :  /&lt;PATH\_TO&gt;/geoip2-csv-converter-v1.1.0
+
+    -   After executing the script, you will find the `final.csv` file inside your current directory.
```
Comment (gomathyk) [inline en/docs/learn/creating-geo-location-dataset.md:30]:
```suggestion
    -   After executing the script, you can find the `final.csv` file inside your current directory.
```

File: en/docs/learn/enabling-geolocation-based-statistics.md [hunk 11]
```diff
@@ -80,12 +78,12 @@ the regions of the users authenticated though WSO2 Identity Server.
         Provide the mysql password when
         prompted.
 
-5.  Download a JDBC provider depending on the database you are using
+4.  Download a JDBC provider depending on the database you are using
     (MySQL in this example) from
     [here](https://www.mysql.com/products/connector/), and extract it.
-6.  Copy the `          mysql-connector-java-<VERSION>.jar         ` to
+5.  Copy the `          mysql-connector-java-<VERSION>.jar         ` to
```
Comment (gomathyk) [inline en/docs/learn/enabling-geolocation-based-statistics.md:84]:
```suggestion
5.  Copy the `          mysql-connector-java-<VERSION>.jar         ` file to
```

File: en/docs/learn/enabling-geolocation-based-statistics.md [hunk 12]
```diff
@@ -80,12 +78,12 @@ the regions of the users authenticated though WSO2 Identity Server.
...
     `          <IS_ANALYTICS_HOME>/lib         ` directory.
```
Comment (gomathyk) [inline en/docs/learn/enabling-geolocation-based-statistics.md:85]:
```suggestion
    the `          <IS_ANALYTICS_HOME>/lib         ` directory.
```

File: en/docs/learn/enabling-geolocation-based-statistics.md [hunk 13]
```diff
@@ -80,12 +78,12 @@ the regions of the users authenticated though WSO2 Identity Server.
...
-7.  Configure the following in
+6.  Configure the following in
```
Comment (gomathyk) [inline en/docs/learn/enabling-geolocation-based-statistics.md:86]:
```suggestion
6.  Configure the following in the
```

File: en/mkdocs.yml [hunk 14]
```diff
@@ -467,6 +467,7 @@ nav:
         - 'Analytics': learn/analytics.md
         - 'Prerequisites to Publish Statistics':
           - 'Prerequisites to Publish Statistics': learn/prerequisites-to-publish-statistics.md
+          - 'Creating Geo Location Data Set' : learn/creating-geo-location-dataset.md
```
Comment (gomathyk) [inline en/mkdocs.yml:470]:
```suggestion
          - 'Creating Geo Location Data Set': learn/creating-geo-location-dataset.md
```

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/learn/creating-geo-location-dataset.md (hunk 8 above)
Comment (ChamathNS) [reply en/docs/learn/creating-geo-location-dataset.md]: Addressed with https://github.com/wso2/docs-is/pull/2716/commits/005760e83eea6c18412d91e41b204fc75bf066fa

File: en/docs/learn/enabling-geolocation-based-statistics.md (hunk 1 above)
Comment (ChamathNS) [reply en/docs/learn/enabling-geolocation-based-statistics.md]: Updated the order with https://github.com/wso2/docs-is/pull/2716/commits/e984f4891204e792e64f06f501641fb304d8a9fe

**Review by gomathyk — APPROVED** [review]

**Pull request merged**